tests/jobs/test_scheduler_job.py

```diff
@@ -207,17 +207,18 @@ def test_process_executor_events(self, mock_stats_incr, mock_task_callback, dag_
         session.commit()
 
         executor.event_buffer[ti1.key] = State.FAILED, None
-
         self.scheduler_job._process_executor_events(session=session)
         ti1.refresh_from_db()
-        assert ti1.state == State.FAILED
+
+        assert ti1.state == State.QUEUED
         mock_task_callback.assert_called_once_with(
             full_filepath='/test_path1/',
             simple_task_instance=mock.ANY,
             msg='Executor reports task instance '
             '<TaskInstance: test_process_executor_events.dummy_task 2016-01-01 00:00:00+00:00 [queued]> '
             'finished (failed) although the task says its queued. (Info: None) '
             'Was the task killed externally?',
+            task=task1,
```

Comment:
I don't know why the` mock_task_callback` `simple_task_instance` is no longer called with `mock.ANY` but with the real `SimpleTaskInstance` object, causing the test to fail. Any ideas?